spec/models/content_file_spec.rb

```diff
@@ -4,27 +4,35 @@
 
 require_relative "../helpers.rb"
 require_relative "../../lib/senkyoshi/models/content"
+require_relative "../mocks/mockzip"
 
 describe "ContentFile" do
   it "should extract data from xml" do
     xml = get_fixture_xml "file.xml"
     file = Senkyoshi::ContentFile.new(xml.xpath("//FILE"))
 
     assert_equal(file.id, "_2041185_1")
-    assert_equal(file.name, "/xid-9066097_2")
+    assert_equal(file.name, "xid-9066097_2/")
     assert_equal(file.linkname, "ADV &amp; DisAdv.pdf")
   end
 
   it "should implement canvas_conversion" do
     xml = get_fixture_xml "file.xml"
     file = Senkyoshi::ContentFile.new(xml.xpath("//FILE"))
+    mock_entry = MockZip::MockEntry.new("ADV &amp;amp; DisAdv.pdf")
     assert_includes(
-      file.canvas_conversion,
+      file.canvas_conversion(mock_entry),
       "href=\"$IMS_CC_FILEBASE$/#{IMPORTED_FILES_DIRNAME}/ADV",
     )
     assert_includes(
-      file.canvas_conversion,
-      "ADV &amp; DisAdv.pdf",
+      file.canvas_conversion(mock_entry),
+      "ADV &amp;amp; DisAdv.pdf",
     )
   end
+
+  it "should strip only leading slash from name" do
+    xml = get_fixture_xml "file.xml"
+    file = Senkyoshi::ContentFile.new(xml.xpath("//FILE"))
+    assert_equal(file.name.start_with?("/"), false)
+  end
```

Comment:
I think it'd be better to use `refute(file.name.start_with?("/"))` over `assert_equal`.

Also, the description of this test says "should strip _only_ leading slash from name", but the spec doesn't actually test that it only removes the leading slash. It just tests that the leading slash was removed. I would change this spec's description or update it so that it tests that other slashes are not removed.